package.json

```diff
@@ -14,7 +14,7 @@
   "module": "./web/noir_wasm.js",
   "sideEffects": false,
   "peerDependencies": {
-    "@noir-lang/noir-source-resolver": "1.1.1"
+    "@noir-lang/noir-source-resolver": "1.1.2"
```

Comment:
This change needs to get done here
https://github.com/noir-lang/noir/blob/master/crates/wasm/build-wasm#L21

Reply:
Need write access to github.com/noir-lang/noir